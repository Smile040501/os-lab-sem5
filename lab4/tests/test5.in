# Test 5
# ./lab -asdmts

120

3 3
101,102,103,
104,105,106,
107,108,109,

44

10

111

3 3
0,0,0,
0,0,0,
0,0,0,