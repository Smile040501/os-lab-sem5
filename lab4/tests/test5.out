3 3
1887,1998,1998,
1887,1998,1998,
1887,1998,1998,
